cmd/root_command.go

```diff
@@ -108,6 +108,9 @@ var (
 			redirectURLFlag, _ := cmd.Flags().GetString("url")
 			if redirectURLFlag != "" {
 				redirectURL = redirectURLFlag
+			} else {
+				pterm.Error.Println("No redirect URL provided. Please provide a URL to redirect API traffic to using the --url or -u flags. This is required to properly run wiretap.")
```

Comment:
This code was removed to allow the dropping of the -u flag for mock mode. `wiretap` should run in mock mode without needing a -u flag, this is the real issue.